angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.spec.ts

```diff
@@ -110,16 +160,16 @@ describe('SprkDropdownComponent', () => {
     fixture.detectChanges();
     dropdownTriggerElement.click();
     fixture.detectChanges();
-    expect(dropDownComponent.isOpen).toEqual(true);
+    expect(dropdownComponent.isOpen).toEqual(true);
     const listElement = fixture.nativeElement.querySelectorAll('li')[0];
     listElement.dispatchEvent(new Event('click'));
     fixture.detectChanges();
-    expect(dropDownComponent.choices[0]['active']).toEqual(true);
+    expect(dropdownComponent.choices[0]['active']).toEqual(true);
   });
 
   it('should set active on click of a choice on a base dropdown if active isnt defined initially', () => {
```

Comment:
Should this test be for a base dropdown or an informational? Update the test name or the test itself.